src/main/java/org/springframework/social/salesforce/api/Salesforce.java

```diff
@@ -39,6 +39,8 @@
 
     public UserOperations userOperations();
 
+    public UserOperations userOperations(String gatewayUrl);
```

Comment:
I see now that it follows the precedence of the methods, seems like a bit of a strange pattern though.

Will merge shortly.